packages/block-library/src/image/edit.native.js

```diff
@@ -218,6 +219,8 @@ export class ImageEdit extends React.Component {
 		const { attributes, isSelected } = this.props;
 		const { url, height, width, alt, href, id, linkTarget } = attributes;
 
+		const actions = [ { label: 'Clear All Settings', onPress: this.onClearSettings, labelStyle: styles.clearSettingsButton } ];
```

Comment:
even though 'Clear All Settings' is localized inside the component I think this is bringing some confusion, I'd prefer to see it is localized here.

Reply:
Will refactor it however, I think it's safer to perform the `localizing` inside the component, just to avoid the case when someone will forget about it.